include/mata/nfa.hh

```diff
@@ -526,6 +558,11 @@ public:
      */
     void remove_epsilon(Symbol epsilon = EPSILON);
 
+    /**
+     * @brief In-place concatenation.
```

Comment:
Does it destroy also the automaton in the argument?

Reply:
It cannot destroy the second automaton: the argument is passed as a const reference.

Reply:
okok